client/src/Redux/Thunk/getRoomNumberAsync.js

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+import { setRoomNumber } from "../Action/roomActions";
+
+export function getRoomNumberAsync(socketId) {
+    return async (dispatch, getState) => {
+            try {
+                const response = await axios({
+                    method: 'post',
+                    url: ('http://localhost:5001') + '/createGame',
```

Comment:
does this work? shouldn't this be our backend url

Reply:
Did you mean the deployed url? If not, the backend URL works. I am still a bit confused what do you mean?

Reply:
like its supposed to be here

https://github.com/VexMihir/chessApp/blob/new-branch/client/src/Redux/Thunk/getRoomNoAsync.js

Reply:
Do you want me to do the time control since the timer is fixed 300 seconds?

Reply:
wdym?

Reply:
There is a big problem. If the player does not finish the game and gets disconnected from the server and then the timer interval won't end and keep decrementing by 1. The current temp solution I can think of is clicking on the browser refresh button.

Reply:
There is a big problem. If the player does not finish the game and gets disconnected from the server and then the setInterval() won't end and keep decrementing by 1. If the user enters the next game, the previous timer that did not end from the last game and the new setInterval() will interrupt the previous setInterval(), The current temp solution I can think of is clicking on the browser refresh button.